src/gnssro/gnss_tec_bufr2ioda.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/env python3
+
+#
+# (C) Copyright 2019-2021 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+from __future__ import print_function
+import argparse
+from datetime import datetime
+import dateutil.parser
+from concurrent.futures import ProcessPoolExecutor
+import numpy as np
+import os
+from itertools import repeat
+import netCDF4 as nc
+
+import pyiodaconv.ioda_conv_engines as iconv
+from pyiodaconv.def_jedi_utils import epoch, iso8601_string, ioda_int_type, ioda_float_type, concat_obs_dict
+from pyiodaconv.orddicts import DefaultOrderedDict
+
+float_missing_value = iconv.get_default_fill_val(np.float32)
+int_missing_value = iconv.get_default_fill_val(np.int32)
+long_missing_value = iconv.get_default_fill_val(np.int64)
+
+locationKeyList = [
+    ('elevationAngleGNSS', 'float'),
+    ('latitude', 'float'),
+    ('longitude', 'float'),
+    ('xECFPosition', 'float'),
```

Comment:
```suggestion
    ('xECEFPosition', 'float'),
```

when I look up this coordinate system `ECEF` is more commonly used, should we adopt this as the standard?